### PR TITLE
GAT-1834: Splitting out references

### PR DESCRIPTION
**Pull request by kymmeh**

### CODE REVIEW

**Review by dannitahdruk — APPROVED** [review]

File: src/modules/AccountTeamMembers/AccountTeamMembers.test.js [hunk 1]
```diff
@@ -33,33 +32,33 @@ describe('Given the AccountTeamMembers component', () => {
                 userState: mocks.userState.mockUserStateNonManager,
             });
 
-            wrapper = render(<AccountTeamMembers {...props} />);
+            wrapper = testUtils.render(<AccountTeamMembers {...props} />);
         });
 
         afterEach(() => {
-            cleanup();
+            testUtils.waitFor();
```
Comment (kymmeh) [inline src/modules/AccountTeamMembers/AccountTeamMembers.test.js:39]: Revert

File: src/modules/AccountTeamMembers/AccountTeamMembers.test.js [hunk 2]
```diff
@@ -96,23 +95,23 @@ describe('Given the AccountTeamMembers component', () => {
...
-            const addMembersButton = screen.getByText(/Add a new member/);
+            const addMembersButton = testUtils.screen.getByText(/Add a new member/);
 
-            fireEvent.click(addMembersButton);
+            testUtils.fireEvent.click(addMembersButton);
         });
 
         afterEach(() => {
-            cleanup();
+            testUtils.waitFor();
```
Comment (kymmeh) [inline src/modules/AccountTeamMembers/AccountTeamMembers.test.js:110]: Revert

File: src/modules/AccountTeamMembers/AccountTeamMembers.test.js [hunk 3]
```diff
@@ -122,49 +121,49 @@ describe('Given the AccountTeamMembers component', () => {
...
```
Comment (kymmeh) [inline src/modules/AccountTeamMembers/AccountTeamMembers.test.js:136]: Revert

File: src/pages/DataAccessRequest/components/UpdateRequestModal/UpdateRequestModal.test.js [hunk 4]
```diff
@@ -1,7 +1,7 @@
 import moxios from 'moxios';
 import { testUtils } from '../../../../../test';
 import UpdateRequestModal from './UpdateRequestModal';
-import { updateRequestProps } from '../../../../utils/__mocks__/darHelperUtils.mock';
+import { updateRequestProps } from '../../../../utils/__mocks__/DarHelper.mock';
```
Comment (kymmeh) [inline src/pages/DataAccessRequest/components/UpdateRequestModal/UpdateRequestModal.test.js:4]: Revert

File: src/pages/DatasetOnboarding/components/UpdateRequestModal/UpdateRequestModal.test.js [hunk 5]
```diff
@@ -1,7 +1,7 @@
 import moxios from 'moxios';
 import { testUtils } from '../../../../../test';
 import UpdateRequestModal from './UpdateRequestModal';
-import { updateRequestProps } from '../../../../utils/__mocks__/darHelperUtils.mock';
+import { updateRequestProps } from '../../../../utils/__mocks__/DarHelper.mock';
```
Comment (kymmeh) [inline src/pages/DatasetOnboarding/components/UpdateRequestModal/UpdateRequestModal.test.js:4]: Revert

File: src/utils/DarHelper.util.test.js [hunk 6]
```diff
@@ -1,9 +1,9 @@
-import DarHelper from './darHelperUtils.util';
-import { formSchema, uniqueSchema, uniquePages, uniqueFormPanels } from './__mocks__/darHelperUtils.mock';
+import { removeStaticPages } from './DarHelper.util';
```
Comment (kymmeh) [inline src/utils/DarHelper.util.test.js:1]: Revert both

**Review by loki-sinclair-hdruk — APPROVED** [review]

**Pull request merged**